### ISSUE TITLE
Auto install script

### ISSUE THREAD

**Issue by marcos10pc** [tp-link-extender/MercuryCore#352]
### Describe the problem

many uers have problems installing mercury core 

### Describe the proposed solution

A bash script that checks for the requirements and auto install the docker and etc

**Comment by ui0ppk** [comment]
wouldnt this have to be like windows linux and mac specific scripts? linux has many distros and package managers so thats going to be hard

**Comment by dxfiscool** [comment]
I don’t believe something like this is necessary. By installing this following the documentation, you are learning how to use Mercury Core and you will make it easier for yourself to be able to edit and use the code.

**Comment by ui0ppk** [comment]
but what if i wanna just try it out quickly?

**Comment by Heliodex** [comment]
> wouldnt this have to be like windows linux and mac specific scripts? linux has many distros and package managers so thats going to be hard

I suppose only an Windows script (powershell, .ps1) and a Unix-like script (bash, .sh) should suffice? I'm unsure about any other scripting utilities that come preinstalled on Windows, Mac, and Linux distros.

**Comment by Bradisverycool** [comment]
I will work on that. While you wait. Bake a cake to end time
